pkg/codegen/jenny_schemareg.go

```diff
@@ -0,0 +1,88 @@
+package codegen
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing/fstest"
+
+	"github.com/grafana/codejen"
+	"github.com/grafana/grafana/pkg/cuectx"
+	"github.com/grafana/kindsys"
+	"github.com/grafana/thema"
+)
+
+// SchemaRegistryJenny generates lineage files into the "next" folder
+// of the local schema registry.
+func SchemaRegistryJenny(path string, latestRegistryDir string) OneToOne {
+	return &schemaregjenny{
+		path:              path,
+		latestRegistryDir: latestRegistryDir,
+	}
+}
+
+type schemaregjenny struct {
+	path              string
+	latestRegistryDir string
+}
+
+func (j *schemaregjenny) JennyName() string {
+	return "SchemaRegistryJenny"
+}
+
+func (j *schemaregjenny) Generate(kind kindsys.Kind) (*codejen.File, error) {
+	name := kind.Props().Common().MachineName
+	oldKindString, err := GetPublishedKind(name, "core", j.latestRegistryDir)
```

Comment:
Does this mean that every execution of `make cue` will require a Git lookup (HTTP request or similar) per existing Kind? 🤔  I'm wondering if this could be a problem in terms of DX?

Reply:
I also realized that this doesn't work without a GH token as it reaches the GH API rate limit. I updated the kind registry workflow and opened a new PR: https://github.com/grafana/grafana/pull/67515